flow-libs/babel-preset-env.js.flow

```diff
@@ -0,0 +1,19 @@
+// @flow strict-local
+
+declare module '@babel/preset-env' {
+  declare type assertVersionCb = () => boolean;
+
+  declare export type Targets = {
+    browsers?: string | Array<string>,
+    esmodules?: boolean,
+    [string]: string,
+    ...
+  };
+
+  declare export type PresetEnvPlugins = Array<mixed>;
```

Comment:
This is actually a tuple (array of fixed length of 2): https://github.com/babel/babel/blob/e498bee10f0123bb208baa228ce6417542a2c3c4/packages/babel-preset-env/src/index.js#L321